modules/smtp/hm-smtp.php

```diff
@@ -111,13 +111,33 @@ function __construct($conf) {
         else {
             $this->port = 25;
         }
-        if (isset($conf['tls']) && $conf['tls']) {
-            $this->tls = true;
-        }
-        else {
-            $this->tls = false;
+        $this->tls = false;
```

Comment:
This seems to be part of another PR (which I already merged). Can you push one change at a time to a PR? Thanks.